src/lib/fixtures.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+const models = require('../models')
+
+function Fixtures (db, config) {
```

Comment:
The definitions of "fixtures" that I can find all refer to testing. It's probably best to use another name to avoid confusion.